### PR TITLE
feat: sqlness test for interval type

### PR DESCRIPTION
**Pull request by QuenKar**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. remove several cases about `interval type` from `dummy.sql`.
2. add interval type `sqlness-test` in `interval.sql`.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: tests/cases/standalone/common/types/interval/interval.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+-- Interval type does not support aggregation functions.
+SELECT MIN(interval_value) from intervals;
+
+SELECT MAX(interval_value) from intervals;
+
+SELECT SUM(interval_value) from intervals;
+
+SELECT AVG(interval_value) from intervals;
+
+DROP TABLE intervals;
```
Comment (killme2008) [inline tests/cases/standalone/common/types/interval/interval.sql:80]: Needs a newline at the end of file

File: tests/cases/standalone/common/types/interval/interval.sql [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+-- ts + interval
+SELECT ts + interval_value as new_value from intervals;
+
+-- ts - interval
+SELECT ts - interval_value as new_value from intervals;
+
+-- DATE + INTERVAL
+SELECT DATE '2000-10-30' + interval_value from intervals;
+
+-- DATE - INTERVAL
```
Comment (killme2008) [inline tests/cases/standalone/common/types/interval/interval.sql:60]: Can we create an issue to track it?

**Review by killme2008 — COMMENTED** [review]
Good job!

@waynexia Please take a look

**Review by waynexia — APPROVED** [review]
LGTM

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**